traffic_ops/traffic_ops_golang/topology/topologies.go

```diff
@@ -572,10 +574,11 @@ func (topology *TOTopology) addParents() (error, error, int) {
 	return nil, nil, http.StatusOK
 }
 
-func (topology *TOTopology) setDescription() (error, error, int) {
-	rows, err := topology.ReqInfo.Tx.Query(updateQuery(), topology.Description, topology.Name)
+func (topology *TOTopology) setNameDescription() (error, error, int) {
```

Comment:
I think we should just change the name of this function to `setTopologyDetails` or something on those lines.

Reply:
Good idea!